backend/handlers/institution_children_request_collection_handler.py

```diff
@@ -52,11 +53,22 @@ def post(self, user, institution_key):
             EntityException
         )
 
+        parent_key = data.get('institution_key')
+        parent_key = ndb.Key(urlsafe=parent_key)
+        requested_inst_key = data.get('institution_requested_key')
+        requested_inst_key = ndb.Key(urlsafe=requested_inst_key)
+        
+        Utils._assert(
+            Institution.has_connection_between(parent_key, requested_inst_key),
```

Comment:
here you pass parent and child, but the order in the function is the opposite
def has_connection_between(possible_child_key, possible_parent_key):

Reply:
Here, the logic is similar to what I explained for the institution_parent_request_collection_handler.